### DISCUSSION TITLE
Implement Symfony clear cache via controller

### DISCUSSION THREAD

**Discussion by stb-mateusz** [symfony/symfony#45731] [Q&A]
Hello,

I have writen aplication in Symfony 5.3.15 and i need to implement clear cache via controller.
I find this [topic][1] but this solution does not work.. :/ when i go to domain.com/admin/clear-cache i have "ERR_CONNECTION_RESET"
I need this, because i have many actions like change default lang which need to recompile container parameters.

I have no idea how to do it

Best regards

  [1]: https://stackoverflow.com/a/53447000/18455682

**Comment by chyshyya** [comment]
It's web server error. Not symfony.
Do you use Apache?

**Reply by chyshyya** [reply]
If so. Try to play with parameters:
```
<IfModule mpm_winnt_module>
    ThreadsPerChild        250
    MaxConnectionsPerChild   0
    ThreadStackSize 8388608
</IfModule>
```
